msteams-ui-styles-core/src/components/input.ts

```diff
@@ -125,7 +127,7 @@ function base(context: IContext, colors: IInputColors) {
           '&:focus': {
             borderBottomColor: colors.active.underline,
             background: colors.focus.background,
-            color: Colors.light.black,
+            color: textColor,
```

Comment:
This should be colors.focus.text and the text color should be set appropriately per theme